db/migrations/mysql/000076_upgrade_lastrootpostat.up.sql

```diff
@@ -0,0 +1,48 @@
+CREATE PROCEDURE Migrate_LastRootPostAt_Default ()
+BEGIN
+	IF (
+			SELECT COUNT(*) FROM INFORMATION_SCHEMA.COLUMNS
+			WHERE TABLE_NAME = 'Channels'
+			AND TABLE_SCHEMA = DATABASE()
+			AND COLUMN_NAME = 'LastRootPostAt'
+			AND (COLUMN_DEFAULT IS NULL OR COLUMN_DEFAULT != 0)
+		) = 1 THEN
+		ALTER TABLE Channels ALTER COLUMN LastRootPostAt SET DEFAULT 0;
+	END IF;
+END;
+	CALL Migrate_LastRootPostAt_Default ();
+	DROP PROCEDURE IF EXISTS Migrate_LastRootPostAt_Default;
+
+CREATE PROCEDURE Migrate_LastRootPostAt_Fix ()
+BEGIN
+	IF (
+		SELECT COUNT(*)
+		FROM Channels
+		WHERE LastRootPostAt IS NULL
+		LIMIT 1
```

Comment:
We already selecting `Count(*)` is there a need for limiting?

Reply:
Oh, no, let me remove that :)